### ISSUE TITLE
DUP Accessibility Issue when choosing Pass Type

### ISSUE THREAD

**Issue by kgorana** [bcgov/parks-reso-public#276]
**Description**

How to replicate:

* Go to a park on DUP public [BC Parks - Day Use Pass](https://reserve.bcparks.ca/dayuse/registration) 
* Select a Pass Type
* Try to use tab to go to next timeslot
* You will be taken to the next button or the bottom of the page if next is disabled 

**Expected Behavior:**

Tab should take you to the next radio button of the radio button group as per WCAG guidelines.
See: [Rating Radio Group Example](https://www.w3.org/WAI/ARIA/apg/example-index/radio/radio-rating.html) 

While arrow keys still work for navigating this section users who only use the keyboard or who are visually impaired may get lost.

**Acceptance Criteria:**

`AM`/`PM`/`DAY` selection works as described by WCAG guidelines for radio button groups:

**Comment by Christopher-walsh22** [comment]
I have created a fix to allow users to tab into section of radio buttons and used the controls mentioned above.  However I have noticed an issue; We have previously created these custom radio buttons to appear larger and have the BC parks coloring. When a user highlights these radio buttons with tab now you are unable to see the outline of your selection on the buttons this due to the increased background and border of the buttons (to appear blue and white). 

The functionality is there, but we will need to re-address the design of these buttons if necessary to show tab focus.

**Comment by manuji** [comment]
Tested on TEST:Passed
* verified that this issue has been fixed. 
* Moving to Done.

**Comment by manuji** [comment]
Deployed to PROD on 6th May 2026.
